tests/provisioning-manifest-build.yml

```diff
@@ -1,2 +1,22 @@
-# nothing to add
-- sleep: 10
+- addMavenRepository: 'https://store.jahia.com/nexus/content/repositories/jahia-public-app-store@id=JahiaStore'
+- addMavenRepository: 'https://devtools.jahia.com/nexus/content/groups/public/@snapshots@noreleases@id=JahiaSnapshot'
+
+# This modules were added during the switch from using jahia-discovery to jahia-ee
+- installBundle: 
+  - 'mvn:org.jahia.modules/press/3.1.0'
```

Comment:
I think we're missing the digitall module here

Reply:
The tests are passing without it, so I'd be in favor of not adding it except if required by the tests.